README.md

```diff
@@ -1,23 +1,23 @@
-# React Enlightenment
+# React 깨우치기
 
-Written by [Cody Lindley](http://codylindley.com/) sponsored by — [Frontend Masters](https://frontendmasters.com/)
+[Frontend Masters](https://frontendmasters.com/)의 후원으로 [Cody Lindley](http://codylindley.com/)에 의해 작성되었습니다.
 
-Learn React in the terse cookbook style found with previous Enlightenment titles (i.e., [jQuery Enlightenment](http://jqueryenlightenment.com/), [JavaScript Enlightenment](http://javascriptenlightenment.com/), [DOM Enlightenment](http://domenlightenment.com/))
+React를 간결한 요리책처럼 배워보세요. 이런 방식은 이전의 깨우치기 출판물에서도 보실 수 있습니다. (예: [jQuery Enlightenment](http://jqueryenlightenment.com/), [JavaScript Enlightenment](http://javascriptenlightenment.com/), [DOM Enlightenment](http://domenlightenment.com/))
 
 ***
 
-**Read Online At**: 
+**온라인에서 읽기**: 
 
 * [reactenlightenment.com](http://www.reactenlightenment.com/)
 
-**Download a .pdf, .epub, or .mobi file from**:
+**.pdf, .epub, 또는 .mobi 파일로 다운로드하기**:
 
 * [https://www.gitbook.com/book/frontendmasters/react-enlightenment/details](https://www.gitbook.com/book/frontendmasters/react-enlightenment/details)
 
-**Contribute content, suggestions, and fixes on GitHub**:
+**GitHub에서 콘텐츠, 제안, 수정사항으로 기여하기**:
 
 * [https://github.com/FrontendMasters/react-enlightenment](https://github.com/FrontendMasters/react-enlightenment)
 
 ***
 
-<a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/3.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/3.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/3.0/">Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License</a>.
+<a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/3.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/3.0/88x31.png" /></a><br />이 저작물은 <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/3.0/">크리에이티브 커먼즈 저작자표시-비영리-변경금지 3.0 현지 저작권법에 맞게 수정하지 않은 저작권</a>에 따라 이용하실 수 있습니다.
```

Comment:
You can replace the link to this: https://creativecommons.org/licenses/by-nc-nd/3.0/deed.ko I found this at the bottom of the page.